src/renderer/screens/Editor/ImportExportDialog.js

```diff
@@ -81,16 +105,30 @@ export const ImportExportDialog = withSnackbar(props => {
           style={{
             display: "flex",
             flexWrap: "wrap",
-            justifyContent: "flex-end",
+            justifyContent: "space-between",
             alignItems: "center"
           }}
         >
-          <CopyToClipboard text={data} onCopy={onCopySuccess}>
-            <Button color="primary">{i18n.editor.copyToClipboard}</Button>
-          </CopyToClipboard>
-          <Button color="primary" onClick={pasteFromClipboard}>
-            {i18n.editor.pasteFromClipboard}
-          </Button>
+          <div style={{ display: "flex" }}>
```

Comment:
Since the defaults are only available for the Model01, would it be possible to lift this part out, and only include the buttons when the currently selected keyboard is a Model01? Or better yet, only when the layout file exists?